GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.cs

```diff
@@ -50,6 +53,7 @@ protected override void SettingsToPage()
             chkShowCurrentBranchInVisualStudio.Checked = AppSettings.ShowCurrentBranchInVisualStudio;
             _NO_TRANSLATE_DaysToCacheImages.Value = AppSettings.AvatarImageCacheDays;
             ShowAuthorAvatar.Checked = AppSettings.ShowAuthorAvatarInCommitInfo;
```

Comment:
This setting should be renamed (not related to this PR)

Reply:
Sure, np